spec/fortnox/api/types/account_number_spec.rb

```diff
@@ -28,4 +28,16 @@
   context 'when AccountNumber created with a negative number' do
     include_examples 'raises ConstraintError', -1
   end
+
+  context 'when AccountNumber created with an invalid string' do
+    include_examples 'raises ConstraintError', 'foo'
+  end
+
+  context 'when AccountNumber created with valid string' do
+    subject { klass['1234'] }
+
+    it 'casts it to a number' do
```

Comment:
We don't really care HOW it does something here, just that we get the correct result. So this could be:
```ruby
context 'created with valid string' do
  subject { klass['1234'] }
  it { is_expected.to eq 1234 }
end
```
Utilizing the readability of RSpecs one line syntax.

Reply:
Actually if we consider how we do things we can change it to `'created with valid value that responds to to_s'`, since that is actually what we allow.